### ISSUE TITLE
Test Runner: Error messages are swallowed

### ISSUE THREAD

**Issue by hemisphera** [microsoft/navcontainerhelper#1917]
**Describe the issue**
Running tests: if an error is thrown outside of the context of a test codeunit, the error is printed to the host but swallowed up and not forwarded to the PS pipeline. This really only happens if there is something wrong on the test framework level and should never happen in normal test scenarios. In my case it is a lock that blows up my test run but Run-TestsInBcContainer happily continues as if nothing happened.

```
ERROR DIALOG: We could not update the data right now.
Please try again later.

Technical details:
A record in table Test Method Line was locked for use in another session. Administrators can see current locks on the Database Locks page or use telemetry. (Snapshot ID: 4dde9222-48cc-4dea-ba9c-25ee3e58356f)

```
![image](https://user-images.githubusercontent.com/8897936/117304475-12d51980-ae7e-11eb-990a-ea8f1f44f0e3.png)

**Comment by freddydk** [comment]
I will check with the team whether we can safely do this - or error dialogs can be part of the normal flow.

**Comment by freddydk** [comment]
Fix included in 2.0.13.preview
